### DISCUSSION TITLE
Latest Chrome update causing Filament not to initialize?

### DISCUSSION THREAD

**Discussion by BStringhamVRSK** [google/filament#7595] [Q&A]
Is anyone else seeing this when running Filament on Web?
After updating to Chrome Version 122.0.6261.70, the call to **Filament.Engine.Create** fails with this error:

**filament.wasm:0xbd3ad Uncaught (in promise) RuntimeError: null function or function signature mismatch**

We are currently running Filament version **1.47.0**.  I'm going to try updating to the latest version, but I doubt that will change things, given that the only difference between success and failure is the Chrome browser version update.

**Comment by ghost** [comment]
Has same issue. Any updates?

**Reply by BStringhamVRSK** [reply]
We resolved this by updating to Filament version 1.49.3 or later.

**Reply by atcha** [reply]
Hello. I can't find any version above 1.49.2 on npm. Is there a way to publish web package for the last releases on npm  ?

**Comment by poweifeng** [comment]
Answered in issue #7598
